sdks/python/build.gradle

```diff
@@ -207,10 +207,10 @@ task directRunnerIT(dependsOn: 'installGcpTest') {
 //    ./gradlew :beam-sdks-python:portableWordCount -PjobEndpoint=localhost:8099
 //
 task portableWordCount {
-  dependsOn portableWordCountTask('portableWordCountExample', project.hasProperty("streaming"), envdir)
```

Comment:
Where is envdir defined? Or was this broken?

Reply:
This is actually cleanup related to #7163 which you had reviewed. Maybe @markflyhigh can approve the change?

Reply:
I approved it, but I just don't understand if Gradle is ignoring and undefined `envdir` or if it's being defined elsewhere, since #7163 removed its definition.

Reply:
It's no longer used and should have been removed in the referenced PR.